app/assets/stylesheets/hosts.scss

```diff
@@ -35,4 +35,44 @@
       }
     }
   }
-}
\ No newline at end of file
+}
+
+.lookup_key_row {
+  .overridden_message {
+    display: none;
+  }
+  &.overridden {
+    span, input, textarea {
+      text-decoration: line-through
+    }
+    background: #ddd;
+    text-color: #333;
+    .override_button {
+      display: none;
+    }
+    .overridden_message {
+      display: inline;
+      text-decoration: none;
+    }
+  }
+  .btn-sm {
+    margin-top: -8px;
+  }
+}
+
+.lookup_key_override_row {
+  display: none;
+  &.visible {
+    display: table-row;
+  }
+  label {
+    font-weight: normal;
+  }
+  .btn-sm {
+    margin-top: -4px;
+  }
+}
+
+.push-down {
+  margin-bottom: 10px;
+}
```

Comment:
I could see this establishing a bad pattern.  What happens when I need `margin-bottom: 20px;` do I create a `.push-down-2x` class?